.nvmrc

```diff
@@ -1 +1 @@
-lts/dubnium
```

Comment:
I think we should switch to using a number rather than a name. This will use the latest version of node 10, same as the name, but it is much easier to understand.

Reply:
I agree that the codenames aren't great, though, this file is mostly for usage with nvm directly. Having the `lts/` in there sends a signal to people looking that we are on an lts release rather than just a "random" version number.
We previously used `lts/*` but that's a problem because sometimes LTS updates can cause an issue.

On a separate note we're 2 LTS majors behind, though, definitely shouldn't be updated as part of this release.

Reply:
Yes we are 2 LTS versions behind and likely all of our tooling will drop support for v10 in April when it is deprecated, but things will likely still work a bit past April. I would recommend that we move to v16 at that point (or wait for it) so that we won't have to update again for awhile. That is, unless we run into problems with v16. but you are right we should hold off on all that for another discussion/time.

Reply:
node 14 just went into LTS, so, it's what we should update it. 16 won't get to LTS until october 2021 https://nodejs.org/en/about/releases/

Reply:
ahh good point, didn't look up the timeline, but yeah 14 is fine too then.

Reply:
fyi, I still would prefer to keep `lts/dubnium` :)